### PR TITLE
Fixes for development setup

### PR DESCRIPTION
**Pull request by clehner**
- Update install command to include admin templates
- Allow configuring DEBUG in order to serve static files during development.

### CODE REVIEW

**Comment by prikhi** [comment]
Looks like I let the Readme get out of sync with the documentation.

`DEBUG_DJANGO` is unnecessary, you can simply do something like:

``` bash
./manage.py runserver --settings=accounting.settings.local
# Or
export DJANGO_SETTINGS_MODULE=accounting.settings.local
./manage.py runserver
```

See: https://acornaccounting.readthedocs.io/en/develop/contribute.html#development-quickstart

So instead of adding DJANGO_DEBUG, you could change the settings files in the Readme from `base` to `local`.

**Comment by clehner** [comment]
Got it. Using `accounting.settings.local` works. I've replaced that commit on this branch with one to update the readme now.

File: README.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ Create a new virtual environment::
 
 Install the prerequisites::
 
-    $ pip install -r requirements/base.txt
+    $ pip install --no-binary django -r requirements/base.txt -r requirements/local.txt
```
Comment (prikhi) [inline README.rst]: Last thing, you can just replace `base.txt` with `local.txt` since `local` sources `test` which sources `base`.

Comment (clehner) [inline README.rst]: I see. amended that commit now.

**Comment by prikhi** [comment]
Also, you don't have to track time on the redmine tracker but you should record your time on your labor sheet.

**Comment by clehner** [comment]
> Also, you don't have to track time on the redmine tracker but you should record your time on your labor sheet.

Cool. I'm using a time tracking tool with checkins and checkouts. I'll transcribe the output of it to the sheet.

**Pull request merged**